src/utils/environment.js

```diff
@@ -0,0 +1,6 @@
+export const ENV = {
```

Comment:
for future reference, `utils` is an indication of bad architecture decisions. Like big trash for the code that not fit in the app structure.